iron-list.html

```diff
@@ -1058,6 +1063,13 @@
       this.templatize(this._userTemplate, this.mutableData);
     },
 
+    _gridChanged: function(newGrid, oldGrid) {
+      if (typeof oldGrid === 'undefined') return;
+      this.fire('iron-resize');
```

Comment:
`this.notifyResize()` (https://github.com/PolymerElements/iron-resizable-behavior/blob/master/iron-resizable-behavior.html#L81) is what you're looking for.